packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -528,10 +535,8 @@ export class Coachmark extends BaseComponent<ICoachmarkProps, ICoachmarkState> i
         (): void => {
           // Need setTimeout to trigger narrator
           this._async.setTimeout(() => {
-            if (this.props.teachingBubbleRef) {
-              this.props.teachingBubbleRef.focus();
-            }
-          }, 500);
+            focusFirstChild(this._entityInnerHostElement.current!);
```

Comment:
What would happen if `current` is `undefined`? Since this is an async timeout I'm wondering if it should check like line 340 does.

Reply:
Fixed